README.md

```diff
@@ -25,22 +25,25 @@ You can build the SD card image locally with Vagrant.
 
 ### Setting up build environment
 
-Make sure you have [vagrant](https://docs.vagrantup.com/v2/installation/) and [docker-machine](https://docs.docker.com/machine/install-machine/) installed.
-Then run the following command to create the Vagrant box and the Docker Machine
-connection. The Vagrant box is needed as a vanilla boot2docker VM is not able to
-run guestfish inside. Use `export VAGRANT_DEFAULT_PROVIDER=virtualbox` to
-strictly create a VirtualBox VM.
+Make sure you have [vagrant](https://docs.vagrantup.com/v2/installation/) installed.
+Then run the following command to create the Vagrant box and use the Vagrant Docker
+daemon. The Vagrant box is needed to run guestfish inside.
+Use `export VAGRANT_DEFAULT_PROVIDER=virtualbox` to strictly create a VirtualBox VM.
 
 ```bash
-make docker-machine
+make docker-setup
```

Comment:
The docker-setup can be replaced by `make vagrant` or `vagrant up`. The export in the Makefile has no effect here. 
So you can reduce the next lines to just export DOCKER_HOST manually.

Reply:
👍